usr/sbin/pkcsslotd/slotmgr.c

```diff
@@ -54,7 +54,7 @@ extern BOOL IveDaemonized;
 void DumpSharedMemory(void)
 {
     u_int32 *p;
-    char Buf[PATH_MAX];
+    char Buf[4 * 8 + 4];
```

Comment:
and please rename to "buf"

Reply:
Done, but this file contains a lot of CamelCase variables.  Maybe that is something for a different set of patches.